src/v3/src/transformer/email/emailChallengeTransformer.ts

```diff
@@ -26,29 +25,21 @@ import {
   UISchemaLayout,
   UISchemaLayoutType,
 } from '../../types';
-import { getUIElementWithName, removeUIElementWithName } from '../utils';
+import { getUIElementWithName } from '../utils';
 
-const TARGET_ELEMENT_KEY = 'credentials.passcode';
-
-export const transformEmailChallenge: IdxStepTransformer = (transaction, formBag, widgetProps) => {
-  const { nextStep, availableSteps } = transaction;
+export const transformEmailChallenge: IdxStepTransformer = ({
+  transaction,
+  formBag,
+  widgetProps,
+}) => {
+  const { nextStep = {} as NextStep, availableSteps } = transaction;
   const { uischema } = formBag;
   const { authClient } = widgetProps;
 
-  const passcodeElement = getUIElementWithName(
-    TARGET_ELEMENT_KEY,
-    uischema.elements as UISchemaElement[],
-  );
-  uischema.elements = removeUIElementWithName(
-    TARGET_ELEMENT_KEY,
-    uischema.elements as UISchemaElement[],
-  );
-
   let reminderElement: ReminderElement | undefined;
-  let verificationCodeElement: FieldElement | undefined;
 
   const resendStep = availableSteps?.find(({ name }) => name?.endsWith('resend'));
-  if (nextStep.canResend && resendStep) {
```

Comment:
Is there ever a scenario where `canResend` will be false but the step exists in available steps?

Reply:
not exist in GenericRemediator

Reply:
Are you sure? I feel like I have seen it while debugging, but maybe I am wrong.